components/skus/browser/rs/cxx/src/lib.rs

```diff
@@ -444,7 +457,7 @@ async fn refresh_order_task(
         .and_then(|order| serde_json::to_string(&order).map_err(|e| e.into()))
         .map_err(|e| e.into())
     {
-        Ok(order) => callback.pin_mut().RunWithResponse(ffi::SkusResult::Ok, &order),
+        Ok(order) => callback.pin_mut().RunWithResponse(ffi::SkusResult::new(ffi::SkusResultCode::Ok, ""), &order),
```

Comment:
Might be worth adding an `ffi::SkusResult::new_ok()` method to cut down on the duplication some.